plutus-tx-plugin/test/Budget/9.6/not-not.uplc.golden

```diff
@@ -4,10 +4,9 @@
        force
          (case
             (force
-               (force
-                  (force ifThenElse
-                     (lessThanInteger 0 x)
-                     (delay (delay (constr 1 [])))
-                     (delay (delay (constr 0 []))))))
+               (force ifThenElse
+                  (lessThanInteger 0 x)
+                  (delay (constr 1 []))
+                  (delay (constr 0 []))))
```

Comment:
Wait, how is this not optimized?

Reply:
Addressed by https://github.com/IntersectMBO/plutus/pull/7042/commits/68a00d11dff0f6596aa1cfeca1bdfe7bcee473df.